template/build/core/phing/tasks/deploy.xml

```diff
@@ -170,4 +179,9 @@
       </fileset>
     </delete>
   </target>
+
+  <target name="deploy:acsf:init" description="Re-initialize ACSF with the settings.php changes required for artifact.">
+    <chmod file="${deploy.dir}/docroot/sites/default/settings.php" mode="0755" />
+    <exec dir="${deploy.dir}/docroot" command="${drush.bin} --include=${deploy.dir}/docroot/modules/contrib/acsf/acsf_init acsf-init -y" logoutput="true" checkreturn="true"/>
```

Comment:
Will it cause any problems to run this against both the canonical repo and the deployment artifact? Why is it necessary to do in both places?